core/src/main/java/hudson/Functions.java

```diff
@@ -1082,29 +1078,55 @@ public static String getFooterURL() {
      * Like {@link #getSortedDescriptorsForGlobalConfig(Predicate)} but with a constant truth predicate, to include all descriptors.
      */
     public static Collection<Descriptor> getSortedDescriptorsForGlobalConfig() {
-        return getSortedDescriptorsForGlobalConfig(Predicates.<GlobalConfigurationCategory>alwaysTrue());
+        return getSortedDescriptorsForGlobalConfig(descriptor -> true);
     }
 
     /**
      * @deprecated This is rather meaningless.
      */
     @Deprecated
     public static Collection<Descriptor> getSortedDescriptorsForGlobalConfigNoSecurity() {
-        return getSortedDescriptorsForGlobalConfig(Predicates.not(GlobalSecurityConfiguration.FILTER));
+        return getSortedDescriptorsForGlobalConfigUnclassified();
```

Comment:
I think this is wrong (there's also tool configurations). Better to retain the existing behavior and just adapt the `Predicate` as needed.